### PR TITLE
Add pangu-for-raycast extension

### PR DESCRIPTION
**Pull request by beviz**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/pangu-for-raycast/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Add spaces between Chinese and English, number or symbols using [https://github.com/vinta/pangu.js]().
+
+![](assets/example.gif)
+
+## Usage
+
+1. (可选) 在Raycast Extensions 中为命令 `pangu` 配置快捷键。
+   (Optional) Configure a shortcut for command `pangu` in Raycast Extensions.
+2. 使用快捷键或 Raycast 命令入口触发该命令，它将格式化屏幕上选择的文本，如果没有选择或者内容微卡，它将格式化剪贴板中的内容，然后粘贴到当前光标位置。
+   Trigger the command using the shortcut or Raycast. The command will format the text you've selected on your screen, or format the content from your clipboard, then pastes it to the current cursor location.
```
Comment (pernielsentikaer) [inline extensions/pangu-for-raycast/README.md:12]: At the moment, Raycast doesn't support localization and only supports US English.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by beviz** [comment]
Thank you for your approvel!

**Pull request merged**